### ISSUE TITLE
1.13.0 isn't passing (any) torrents to Deluge

### ISSUE THREAD

**Issue by phenomeni** [autobrr/autobrr#612]
**Version**
1.13.0

**Describe the bug**
Opened a bug yesterday thinking this issue was solely relating to TorrentSeeds, but after testing it's universal: no torrents are being passed to the client (Deluge 2.0.5) 

Nothing was changed in 'Settings' between upgrading from the previous version of Autobrr to 1.13.0. All worked fine prior to the upgrade.

**Expected behavior**
Torrents are being matched and Autobrr reports Green Ticks on the dashboard, yet the torrents are not being passed from Autobrr to the torrent client correctly.

If I manually upload a .torrent to the Deluge Watch folder, it's immediately picked up, so  that rules out any issue with Deluge.

Although no settings were changed prior to or after the Autobrr upgrade, going to Autobrr > Settings > Clients and using the 'Test' button returns 'OK'. Again, this shows that the issue is not a Deluge issue.

**Additional context**
I tested the above issue with three different trackers before posting this bug. The issue is only present in Autobrr 1.13.0 and is 100% related to passing the .torrent to the client.

**Comment by s0up4200** [comment]
Hi there.

I can't seem to reproduce your error.

My setup:

- Deluge 2.0.3
- autobrr 1.13.0

I set up a wide open filter with all indexers enabled and created a Deluge 2-action.
10 torrents was added succesfully.

Did you set a label for the Deluge action?
The label needs to exist in Deluge for autobrr to use it. If it doesnt exist, nothing will be added to Deluge.

I recommend you come by Discord if you still need help, as you will get help quicker that way 😃
